src/main/java/net/fabricmc/loader/transformer/FabricTransformer.java

```diff
@@ -56,6 +58,11 @@
 		ClassVisitor visitor = classWriter;
 		int visitorCount = 0;
 
+		if (isMinecraftClass && FabricLoader.INSTANCE.getAccessWidener().getTargets().contains(name)) {
```

Comment:
This check may not be reached outside of dev env. Scroll up; you need to change the condition check for `transformAccess` local var to check for existence in access widener as well!

Reply:
Ah yes, good spot.

Reply:
https://github.com/FabricMC/fabric-loader/pull/206/files#diff-357f0c28bb837e6ad9c13252885f47bcR52 Still needs to be fixed.